aries-site/src/pages/components/card.mdx

```diff
@@ -51,16 +62,86 @@ A card by nature has limited real-estate, so it should present summarized inform
 
 ## When to use
 
+- Presenting high-level, summarized pieces of information.
+- Presenting an item, or collection of items, which can best be consumed visually.
+
+In all cases the presentation allows users to quickly scan the content and navigate to underlying detail if desired.
+
 ## When not to use
 
-- Do not use a Card to filter data on the same screen, instead use **\_**.
+- Cards should not be used as a control to filter data on the same screen. A button is the appropriate control to apply a filter. Alternatively, cards may be used to present summarized information and allow a user to navigate to a pre-filtered view of that information.
+
+<BestPracticeGroup>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'do',
+      message:
+        'Style status filter buttons as default buttons when the buttons are placed on the same page as the data they will be filtering.',
+    }}
+    scale={0.4}
+    width="100%"
+  >
+    <CardFilteringBestPractice />
+  </Example>
+  <Example
+    height={{ min: 'small' }}
+    bestPractice={{
+      type: 'dont',
+      message: `Don’t use a card to wrap a set of status filter buttons that affect data on the same page. Cards should act as independent regions that do not affect surrounding content. However, placing these buttons in a Card would be appropriate on a dashboard, where clicking on one of the status buttons would navigate to a different page that is filtered by the selected status.`,
```

Comment:
I don't quite understand this part "that affect data on the same page" in light of the visual example. Just a note.

Reply:
lowercase "c" in cards, @taysea This should be changed everywhere. I will try to point them out but please do try with me to find them! Like we're looking for Waldo :D

Reply:
I've adjusted the wording to clarify and lowercased the c.